decidim-generators/lib/decidim/generators/install_generator.rb

```diff
@@ -130,19 +130,8 @@ def letter_opener_web
       def recreate_db
         soft_rails "db:environment:set", "db:drop"
         rails "db:create"
-
-        # In order to ensure that migrations don't eager load models with not
-        # yet fully populated schemas (which could break commands which load
-        # migrations and seeds in the same process, such as `rails db:migrate
-        # db:seed`), we make sure to run them in the same process if seeds are
-        # requested so that we can catch these situations earlier than end
-        # users.
-        if options[:seed_db]
-          rails "db:migrate", "db:seed"
-        else
-          rails "db:migrate"
-        end
-
+        rails "db:migrate"
```

Comment:
Why? If you had to change this maybe you forgot to define a class inside a migration.

Reply:
In the migration I need to access elements defined in the ContentBlock model (I need to access the `images_container` dynamic class). If I redefine the class in the migration then I cannot access that dynamic class, and makes life a lot harder.

I've talked to @deivid-rodriguez and agreed to change this file.